Installation/test/Installation/test_find_package.cmake.in

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION ${CMAKE_VERSION})
+project( test_find_package_${mode} )
+find_package(CGAL ${VERSION} ${EXACT}REQUIRED
```

Comment:
Is there a missing space after EXACT?

Reply:
No: `${EXACT}` evaluates to an empty string, or to `EXACT ` (with a space).

But maybe I should add the space, for readability. That does not change anything.